Notes/Strasnote.Notes.Data.Entities/UserEntity.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Strasnote
+// Licensed under https://strasnote.com/licence
+
+using System.Collections.Generic;
+
+namespace Strasnote.Notes.Data.Entities
+{
+	/// <summary>
+	/// User entity
+	/// </summary>
+	public sealed class UserEntity : Auth.Data.Entities.UserEntity, IEntity
+	{
+		/// <summary>
+		/// User ID (alias for <see cref="Microsoft.AspNetCore.Identity.IdentityUser{TKey}.Id"/>)
+		/// </summary>
+		public long UserId
+		{
+			get => Id;
+			set => Id = value;
+		}
+
+		/// <summary>
+		/// The User's given name (or Christian / first name)
+		/// </summary>
+		public string UserGivenName { get; init; } = string.Empty;
+
+		/// <summary>
+		/// The User's full name
+		/// </summary>
+		public string UserFullName { get; init; } = string.Empty;
```

Comment:
Don't think we need to know this either :)

Reply:
True - if we're being super not taking people's details then all we really need is an email address I suppose!  Although when sharing a note / folder if we send an email to the receiving person it might be helpful to say 'xxx has shared a note with you'?

Reply:
Alright, happy to have this as another optional 'profile' field

Reply:
Should we have a UserProfile entity?

Reply:
Yes good idea 😀

Reply:
OK so I've made a Profile record inside the UserEntity class, which *should* serialise / deserialise as JSON automatically

Reply:
So we'll have a json blob as the user profile?

Reply:
Yup - rather than a separate table.  It seemed mad to have a UserProfileId when User <-> UserProfile is a 1-1 relationship

Reply:
Cool sounds good to me

Reply:
OK great I'll merge this PR then!